packages/cx-navigation/doc/BurgerMenu.md

```diff
@@ -9,28 +10,45 @@ The CX BurgerMenu Component provides a triggerable navigation menu for Mobile an
 - LanguageButton
 - Overlay
 
-The MenuToggler inside the BurgerMenu component is just an icon link to close the burger menu and it is closed by default. For opening it, another icon link has to be added to another component slot, like in the Header from `bodiless/cx-layout` package. Both links components are transformed into a toggler by reusing `asBurgerMenuToggler` HOC to update the React state through `withBurgerMenuProvider`, which needs to be added around the BurgerMenu component, like in a page layout.
+The `MenuToggler` inside the `BurgerMenu` component is an icon link to close the Burger Menu (which
+is closed by default). For opening it, another icon link has to be added to another component slot,
+like in the Header from the `bodiless/vital-layout` package. Both links components are transformed
+into a toggler by reusing `asBurgerMenuToggler` HOC to update the React state through
+`withBurgerMenuProvider`, which needs to be added around the `BurgerMenu` component, like in a page
+layout.
 
-The Main Menu content is the same as the Top Navigation Menu on Desktop. We also apply the same WhereToBuy, UtilityMenu, and LanguageButton, compoments from Desktop Top Nav, but following a different order for the BurgerMenu.
+The Main Menu content is the same as the Top Navigation Menu on Desktop. We also apply the same
+`WhereToBuy`, `UtilityMenu`, and `LanguageButton` components from Desktop Top Nav, but following a
+different order for the Burger Menu.
 
-The Overlay is an empty div, defined as a sibling for the BurgerMenu wrapper, and it is necessary to provide a background layer for the BurgerMenu as it is opened. That layer is blur and requires some tailwind customizations with filter backdrop to achieve the desired design. By default, it is not possible to see the Overlay on Mobile since the BurgerMenu is full width, but it is visible on Tablet once it takes place only in part of the screen.
+The Overlay is an empty `<div>`, defined as a sibling for the `BurgerMenu` wrapper, and it is
+necessary to provide a background layer for the `BurgerMenu` as it is opened. That layer is blurred
```

Comment:
I would reword this as 
On tablet the burger menu only covers partial portion of the site and site itself is blurred to prevent user interaction with it.  The blur effect is done using Backdrop blur https://tailwindcss.com/docs/backdrop-blur and cutomized to your design requirements.
On mobile, the burger menu uses the full width of the viewport.

Reply:
✅ Updated.